README.md

```diff
@@ -5,15 +5,25 @@
 [![Swift 5.0](https://img.shields.io/badge/Language-Swift_5.0-orange.svg)](https://swift.org)
 ![](https://img.shields.io/badge/Deployment%20Target-iOS%2011.3-blue.svg)
 
-EOSIO SDK for Swift: Vault is a utility library for working with public/private keys and signing with Apple's Keychain and Secure Enclave.
+EOSIO SDK for Swift: Vault consists of two main components; Vault and Vault Signature Provider.  
 
-The Vault library is a required dependency of the [EOSIO SDK for Swift: Vault Signature Provider](https://github.com/EOSIO/eosio-swift-vault-signature-provider). It additionally provides key generation, management and signing functions that can be called directly.
+Vault is a utility library for working with public/private keys and signing with Apple's Keychain and Secure Enclave.
+
+Vault Signature Provider is a pluggable signature provider for [EOSIO SDK for Swift](https://github.com/EOSIO/eosio-swift). It allows for signing transactions using keys stored in Keychain or the device's Secure Enclave.
+
+The Vault component is a required dependency of Vault Signature Provider. It additionally provides key generation, management and signing functions that can be called directly.
 *All product and company names are trademarks™ or registered® trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.*
```

Comment:
Can we add a line break above the italicized text here? When rendered, it's rendering as part of the same paragraph of the sentence that precedes it.

Reply:
Done!